appeals/web/src/server/appeals/appeal-details/issue-decision/issue-decision.mapper.js

```diff
@@ -312,22 +303,46 @@ function checkAndConfirmPageRows(appealData, session) {
 		});
 	}
 
-	const appellantCostDecisionOutcome = session.appellantCostDecision?.outcome;
-	if (appellantCostDecisionOutcome) {
+	const appellantCostsDecisionOutcome = session.appellantCostsDecision?.outcome;
+	if (appellantCostsDecisionOutcome) {
 		rows.push({
 			key: "Do you want to issue the appellant's costs decision?",
-			value: appellantCostDecisionOutcome === 'true' ? 'Yes' : 'No',
+			value: appellantCostsDecisionOutcome === 'true' ? 'Yes' : 'No',
```

Comment:
There's a utility you may want to use here if it's not more trouble than it's worth, `convertFromBooleanToYesNo` in `appeals/web/src/server/lib/boolean-formatter.js`